web_development_101/the_front_end/project_html_css.md

```diff
@@ -1981,6 +1981,7 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [João Fonseca's Solution (Easy)](https://github.com/FonsecaJoao/google-homepage) - [View in Browser](https://fonsecajoao.github.io/google-homepage/)
 * [Kenny Chung's Solution (Easy)](https://github.com/kpchungdev/google_home_page) - [View in Browser](https://kpchungdev.github.io/google_home_page/)
 * [Kenny Chung's Solution (Difficult)](https://github.com/kpchungdev/google_search_results_page) - [View in Browser](https://kpchungdev.github.io/google_search_results_page/)
+* [vanny96's Solution (Easy)](https://github.com/vanny96/google-homepage) - [View in Browser](https://vanny96.github.io/google-homepage/) cacca
```

Comment:
Did you mean to write cacca here?

Reply:
No I didn't meant to :/ you can delete it from the code! (cacca means "shit" in italian... I may have used it to test something and it slipped... I'm sorry :/)